lib/models/database/comment/comment_data.dart

```diff
@@ -0,0 +1,71 @@
+import "package:cloud_firestore/cloud_firestore.dart";
+import "package:flutter/foundation.dart";
+import "package:proxima/models/database/user/user_id_firestore.dart";
+
+@immutable
+class CommentData {
```

Comment:
I think we could add a single line of documentation for each of the data classes. Just to inform the user of the class and overall usage of the class in code editors.

Reply:
You're are right, done in 4c0fe6830a9dc7c25e90fb958efa76ed6b5547a5.